src/code/code_800FCE80.c

```diff
@@ -132,6 +139,9 @@ f32 Math_FAtanF(f32 x) {
     }
 }
 
+/**
+ * @return angle from (1,0) to (x,y) in radians, in (-pi,pi] range
```

Comment:
Sorry for the late response, I must have missed the notification

No it's not a typo, it's refering to the unit circle with (1,0) being at 1 on the x axis (or maybe more simply the angle from the vector (1,0) to the vector (x,y))

The range is `(-pi;pi]` as in, it could be `[0;2pi)` instead (or maybe the angle could be unoriented and it'd be `[0;pi]`), I'm not sure what to say
It's trying to clear up the unit used (radians), it being oriented, and the range of the principal value used here

Reply:
ah, I didn't consider that (1,0) would be an orientation vector. I usually conceptualize atan2 as being the tool you use to find the angle between two points, so I interpreted `angle from (1,0)` to be expressing the point of origin for calculations.

As for the range, was more just asking for clarification for why the range of return values is expressed as `(-pi, pi]`, but I've worked it out to be correct.

Reply:
So is this fine then?

Reply:
i would suggest making it clearer that (1,0) isn't a point

Reply:
like
```suggestion
 * @return angle from vector (1,0) to (x,y) in radians, in (-pi,pi] range
```
?

Reply:
yea, think that's good enough